Upstream-Status: Inappropriate [embedded specific]
==================================================
diff --git a/conf/parsers.conf b/conf/parsers.conf
index 77f61ce03..f5a636a61 100644
--- a/conf/parsers.conf
+++ b/conf/parsers.conf
@@ -106,7 +106,7 @@
     # http://rubular.com/r/tjUt3Awgg4
     Name cri
     Format regex
-    Regex ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<message>.*)$
+    Regex ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<log>.*)$
     Time_Key    time
     Time_Format %Y-%m-%dT%H:%M:%S.%L%z

